### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by thejinx0r**
Minor changes that improve performance (using a single thread for fuse ),
and readability.

### CODE REVIEW

File: src/bindings.cc [hunk 1]
```diff
@@ -93,11 +93,11 @@ namespace NodeFuse {
         }
 
         fuse_session_add_chan(fuse->session, fuse->channel);
-        if(fuse->multithreaded){
-            ret = fuse_session_loop_mt(fuse->session); //blocks here
-        }else{
-            ret = fuse_session_loop(fuse->session); //blocks here            
-        }
+        // if(fuse->multithreaded){
```
Comment (c4milo) [inline src/bindings.cc]: I would just leave this out if it is to be commented.

**Comment by c4milo** [comment]
LGTM

File: src/bindings.cc (hunk 1 above)
Comment (thejinx0r) [inline src/bindings.cc]: Good point.
I wasn't sure if I was going to revert back to threaded fuse, but threading really isn't needed with nodejs.

**Comment by c4milo** [comment]
:+1:

**Pull request merged**